services/QuillLMS/engines/comprehension/app/models/comprehension/prompt.rb

```diff
@@ -11,6 +13,7 @@ class Prompt < ActiveRecord::Base
     has_many :automl_models, inverse_of: :prompt
     has_many :prompts_rules
     has_many :rules, through: :prompts_rules, inverse_of: :prompts
+    has_many :change_logs
```

Comment:
Does this actually work?  `ChangeLog` is a model that isn't inside the Comprehension Engine, which leads me to suspect that this automatic relationship short-cut in Rails wouldn't work.